src/main/java/school/lemon/changerequest/java/introduction/pr1/ConsoleCalculator.java

```diff
@@ -1,9 +1,82 @@
 package school.lemon.changerequest.java.introduction.pr1;
 
 
+import java.util.Scanner;
+
 public class ConsoleCalculator {
 
     public static void main(String[] args) {
+        listOfChoises();
+        Scanner sc = new Scanner(System.in);
+        String operation;
+        while (!sc.hasNext("(exit|help|div|mul|sub|add)")) {
+            System.out.println("False input, try again");
+            listOfChoises();
+            sc.next();
+        }
+        while (true) {
+            operation = sc.next().toLowerCase();
+            if (operation.equals("exit")) {
+                System.out.println("Bye-bye");
+                sc.close();
+                break;
+            } else if (operation.equals("help")) {
+                listOfChoises();
+                sc.next();
+            }
+            System.out.println("Enter first number:");
+            int num1;
+            while (!sc.hasNextInt()) {
+                System.out.println("Not a DIGIT!");
+                sc.next();
+            }
+            num1 = sc.nextInt();
+            sc.nextLine();
+            System.out.println("Enter second number:");
+            int num2;
+            while (!sc.hasNextInt()) {
```

Comment:
Duplicated loops. Please extract reading numbers to separate  method. Should be like:
`int num2 = readNextInt()`;